runtime/doc/eval.txt

```diff
@@ -9939,8 +9939,8 @@ This does NOT work: >
 			variables are automatically deleted when the function
 			ends.
 
-:unl[et] ${env-name}				*:let-environment* *:let-$*
-			Rermove environment variable {env-name}.
+:unl[et] ${env-name}
```

Comment:
Oh, did you totally remove the tags?
I thought they should be: `*:unlet-environment* *:unlet-$*`.